src/services/prkset/VehicleLookupService.ts

```diff
@@ -1,56 +1,63 @@
-import { makeGet } from '../util';
+import { JsonApiClient } from 'services/client/JsonApiClient';
+import { ax } from '../util';
 
 export class VehicleLookupService {
   static cached: VehicleParts<VehicleLookupPayload[]>;
 
   static getElectricals = async () => {
-    let entities: VehicleLookupPayload[] = await makeGet('/api/v1/vehicle_electricals');
-    return entities;
+    const client = clientGenerator(VehiclePath.Electricals);
+    const result = await client.find();
+    return result;
   }
 
   static getTypes = async () => {
-    let entities: VehicleLookupPayload[] = await makeGet('/api/v1/vehicle_types');
-    return entities;
+    const client = clientGenerator(VehiclePath.Types);
+    const result = await client.find();
+    return result;
   }
 
   static getLengthRanges = async () => {
-    let entities: VehicleLookupPayload[] = await makeGet('/api/v1/vehicle_length_ranges');
-    return entities;
+    const client = clientGenerator(VehiclePath.LengthRanges);
+    const result = await client.find();
+    return result;
   }
 
   static getSlides = async () => {
-    let entities: VehicleLookupPayload[] = await makeGet('/api/v1/vehicle_slides');
-    return entities;
+    const client = clientGenerator(VehiclePath.Slides);
+    const result = await client.find();
+    return result;
   }
 
   static getTowings = async () => {
-    let entities: VehicleLookupPayload[] = await makeGet('/api/v1/vehicle_towings');
-    return entities;
+    const client = clientGenerator(VehiclePath.Towings);
+    const result = await client.find();
+    return result;
   }
 
+
   static getLookups = async () => {
-    if (VehicleLookupService.cached) {
-      return VehicleLookupService.cached;
+    if (this.cached) {
+      return this.cached;
     }
 
     const requests = {
-      types: VehicleLookupService.getTypes(),
-      lengthRanges: VehicleLookupService.getLengthRanges(),
-      slides: VehicleLookupService.getSlides(),
-      towings: VehicleLookupService.getTowings(),
-      electricals: VehicleLookupService.getElectricals(),
+      types: this.getTypes(),
+      lengthRanges: this.getLengthRanges(),
+      slides: this.getSlides(),
+      towings: this.getTowings(),
+      electricals: this.getElectricals(),
```

Comment:
Acá no va `this` xq getLookups es static, todo es static...
Entonces hay q usarlo como estaba